### PR TITLE
Kueue intro tutorial yaml update

### PR DESCRIPTION
**Pull request by rbarberop**
Updated and tested yaml to v1beta1

### CODE REVIEW

**Comment by rbarberop** [comment]
FYI @alculquicondor

**Review by alculquicondor — COMMENTED** [review]
lgtm

File: batch/kueue-intro/cluster-queue.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
```
[the remark this answers is not in the document]
Comment (rbarberop) [reply batch/kueue-intro/cluster-queue.yaml]: Rolled back in all files

**Review by bourgeoisor — APPROVED** [review]
LGTM

**Comment by alculquicondor** [comment]
blocked on kubernetes-sigs/kueue#659

**Comment by rbarberop** [comment]
@bourgeoisor all the blocks have been lifted, this is ready to merge.

**Review by bourgeoisor — APPROVED** [review]

**Pull request merged**